templates/vue/package.json

```diff
@@ -41,7 +43,7 @@
     "cross-env": "^5.0.5",
     "css-loader": "^0.28.7",
     "cypress": "^3.8.0",
-    "eslint": "^6.5.1",
+    "eslint": "^5.16.0",
```

Comment:
This looks like we're downgrading to me. Could you try updating this to `7.1.0` and see if that works? We have a pending PR at the moment upgrading the eslint version to that version.